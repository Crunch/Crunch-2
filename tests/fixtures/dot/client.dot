<div>Hi {{=it.name}}!</div>
<div>{{=it.age || ''}}</div>