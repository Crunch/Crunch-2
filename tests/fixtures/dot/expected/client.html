function anonymous(it
/**/) {
var out='<div>Hi '+(it.name)+'!</div><div>'+(it.age || '')+'</div>';return out;
}